src/spec/orderexport.spec.coffee

```diff
@@ -109,6 +112,7 @@ describe 'OrderExport', ->
 
     @orderExport.runCSVAndStreamToFile(callback)
       .then () =>
+        expect(data).toEqual([ 'id,customerEmail', 'aaa,\nbbb,\nccc,' ])
```

Comment:
Would this log the header and the first row on the same line? I'm unable to test this with headers (for some strange reasons)